package.json

```diff
@@ -4,64 +4,50 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "serve": "preact build --no-prerender && preact serve",
+    "build": "preact build",
+    "serve": "sirv build --port 8080 --cors --single",
     "dev": "preact watch",
     "lint": "eslint src",
```

Comment:
I think we probably still want the license-checker. 
```suggestion
    "lint": "eslint src",
    "license": "license-checker --csv --production > licenses.csv"
```
I tested that this still works after introducing this line and the dependency further below.
```
$ npm run license

> architecture-viewer@1.0.0 license /Users/dj/Code/architecture-viewer-swaraj
> license-checker --csv --production > licenses.csv

$ cat licenses.csv 
"module name","license","repository"
"ajv@6.12.3","MIT","https://github.com/ajv-validator/ajv"
"architecture-viewer@1.0.0","UNLICENSED",""
"cytoscape-dagre@2.2.2","MIT","https://github.com/cytoscape/cytoscape.js-dagre"
"cytoscape-popper@1.0.7","MIT","https://github.com/cytoscape/cytoscape.js-popper"
"cytoscape@3.15.2","MIT","https://github.com/cytoscape/cytoscape.js"
"dagre@0.8.5","MIT","https://github.com/dagrejs/dagre"
"fast-deep-equal@3.1.3","MIT","https://github.com/epoberezkin/fast-deep-equal"
"fast-json-stable-stringify@2.1.0","MIT","https://github.com/epoberezkin/fast-json-stable-stringify"
"graphlib@2.1.8","MIT","https://github.com/dagrejs/graphlib"
"heap@0.2.6","PSF","https://github.com/qiao/heap.js"
"json-schema-traverse@0.4.1","MIT","https://github.com/epoberezkin/json-schema-traverse"
"lodash.debounce@4.0.8","MIT","https://github.com/lodash/lodash"
"lodash@4.17.19","MIT","https://github.com/lodash/lodash"
"popper.js@1.16.1","MIT","https://github.com/FezVrasta/popper.js"
"preact-render-to-string@5.1.10","MIT","https://github.com/developit/preact-render-to-string"
"preact-router@3.2.1","MIT","https://github.com/developit/preact-router"
"preact@10.4.6","MIT","https://github.com/preactjs/preact"
"pretty-format@3.8.0","MIT","https://github.com/thejameskyle/pretty-format"
"punycode@2.1.1","MIT","https://github.com/bestiejs/punycode.js"
"tippy.js@2.5.4","MIT","https://github.com/atomiks/tippyjs"
"uri-js@4.2.2","BSD-2-Clause","https://github.com/garycourt/uri-js"
```